python/kruskal_algo.py

```diff
@@ -0,0 +1,66 @@
+class Graph:
+    def __init__(self, vertices):
```

Comment:
Codacy found an issue: [Missing docstring in __init__ (D107)](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6284228)